File: tresnumeros.m
pkg load database;
clc; 
clear all; 
close all;

n1=input('Introduce el primer numero: ');
n2=input('Introduce el segundo numero: ');
n3=input('Introduce el tercer numero: ');
sum=0;
mul=0;
  if n2<n1&&n3<n1
     fprintf('El primer numero es el mas grande \n');
     sum=n1+n2+n3;
     fprintf('La suma de los numeros es: %d\n' ,sum);
  endif
  
  if n3<n2&&n1<n2
      fprintf('El segundo numero es el mas grande \n');
      mul=n1*n2*n3;
      fprintf('La multiplicacion de los numeros es: %d\n' ,mul);
  endif
  
  if n2<n3&&n1<n3
      fprintf('El tercer numero es el mas grande \n');
      x=mat2str(n1);
      y=mat2str(n2);
      z=mat2str(n3);
      f= strcat(x, y, z);
      fprintf('El concatenado es: %s \n', f);
  endif
  
  if n1==n2&&n1==n3&&n3==n2
     fprintf('Todos los numeros son iguales \n');
     fprintf('El primero num. es: %d\n', n1);
     fprintf('El segundo num. es: %d\n', n2);
     fprintf('El tercero num. es: %d\n', n3);
  endif

%Conectar con PGADMIN
params= cell(1,3);
params{1,1}=n1;
params{1,2}=n2;
params{1,3}=n3;

conn = pq_connect(setdbopts('dbname','tarea1','host','localhost',
'port','5432','user','postgres','password','jblue5queso'));
N=pq_exec_params(conn, "insert into tresnumero(n1,n2,n3) values($1,$2, $3);",params); %insertar datos en la tabla

%Texto
save('tresnumeros.txt','n1','n2', 'n3','-ascii');

File: tresnumeros.py

import psycopg2

def  post(x,y,z):
    try:
        conexion = psycopg2.connect(
            host = "localhost",
            port = "5432",
            user = "postgres",
            password = "jblue5queso",
            dbname = "tarea1"
            )
    except:
        print("Sin Conexion Exitosa\n")
            
    cursor = conexion.cursor()
    cursor.execute("INSERT INTO tresnumero(n1, n2, n3) VALUES (%s, %s, %s);", (x, y, z))
    conexion.commit()
    cursor.close()
    conexion.close()

n1=int(input("Ingrese el primer número: "))
n2=int(input("Ingrese el segundo número: "))
n3=int(input("Ingrese el tercer número: "))

if n2<n1 and n3<n1:
     print("El primer numero es el mas grande \n")
     sum=n1+n2+n3
     print("La suma de los numeros es: " ,sum)
  
if n3<n2 and n1<n2:
      print("El segundo numero es el mas grande \n")
      mul=n1*n2*n3
      print("La multiplicacion de los numeros es: " ,mul)
  
  
if n2<n3 and n1<n3:
      print("El tercer numero es el mas grande \n")
      x=str(n1)
      y=str(n2)
      z=str(n3)
      con=x+y+z
      print("El concatenado es: ", con)
  
  
if n1==n2 and n1==n3 and n3==n2:
    print("Todos los numeros son iguales \n")
    print("El primero num. es: ", n1)
    print("El segundo num. es: ", n2)
    print("El tercero num. es: ", n3)

post(n1, n2, n3)

